addon/adapters/cloud-firestore-modular.ts

```diff
@@ -86,8 +84,9 @@ export default class CloudFirestoreModularAdapter extends Adapter {
     return fastboot && fastboot.isFastBoot;
   }
 
-  public init(...args: unknown[]): void {
-    this._super(...args);
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  public constructor(...args: any[]) {
```

Comment:
Tried using the `unknown[]` type here but I can't then call `super()` with that unknown type.